src/main/haskell/language-kore/test/parser/Data/Kore/Unparser/UnparseTest.hs

```diff
@@ -64,14 +64,15 @@ unparseUnitTests =
             \    )\n\
             \]"
         , unparseTest
-            Module
+            (Module
```

Comment:
Don't you get warnings with these?

Reply:
No, I don't (they are required because of the typing information).